frontend/src/metabase/dashboard/components/DashboardEmbedHeaderButton/DashboardEmbedHeaderButton.tsx

```diff
@@ -39,6 +39,11 @@ export const DashboardEmbedHeaderButton = forwardRef(
       disabled,
     });
 
+    const onHeaderButtonClick = (event: MouseEvent<HTMLButtonElement>) => {
+      event.preventDefault();
+      onClick?.();
+    };
```

Comment:
I'm not sure what this does. Can you elaborate on why we just pass `onClick`?

Reply:
It's a consequence of the the whole disabled button + tooltip thing with Mantine (https://v6.mantine.dev/core/button/#disabled-button-with-tooltip) - we need to explicitly ensure that we precent the default event to make sure that the tooltip still works on disabled buttons

Reply:
I've tried removing `onHeaderButtonClick` and passing `onClick` directly and it worked. By the way, I'm not sure how `onClick` should solve the problem when tooltip doesn't show because tooltip will appear on hover, not on clicked.

Can you double check?

Reply:
Let me try something real quick. I might be able to get rid of the Box

Reply:
I'll readd the Box for now to get tests to pass, Cypress really wants that wrapper div, at least in the CI environment. locally it works fine but I don't want to play with fire here